.github/workflows/ci.yml

```diff
@@ -9,17 +9,28 @@ on:
       - master
 
 jobs:
-  test-ubuntu:
+  test-debian:
     strategy:
       matrix:
-        ubuntu:
-          - 16.04
-          - latest
-    runs-on: ubuntu-${{ matrix.ubuntu }}
+        image:
+          - ubuntu:latest
+          - ubuntu:16.04
+          - debian:testing
+          - debian:stable
+          - debian:oldstable
+    runs-on: ubuntu-latest
+    container:
+      image: docker.io/library/${{ matrix.image }}
     steps:
+      - name: install dependencies
+        run: |
+          apt-get update &&
```

Comment:
I am pretty sure the runner runs ``apt update`` somewhere cause the images do not contain any apt lists by default. Maybe we want to keep it just to be sure.

Reply:
Why is this comment pending?!

Reply:
`apt-get update` is required. The Ubuntu VMs already have the package lists, but the Docker images don't.